### PR TITLE
RP 189 adding zypper to BinDeps updated

### PR DESCRIPTION
**Pull request by colbec**

### CODE REVIEW

File: src/dependencies.jl [hunk 1]
```diff
@@ -199,6 +199,30 @@ pkg_name(p::Pacman) = p.package
...
+type Zypper <: PackageManager
+    package::AbstractString
+end
+can_use(::Type{Zypper}) = has_zypper && OS_NAME == :Linux
+package_available(z::Zypper) = can_use(Zypper) && success(`zypper se $(z.package)`)
+function available_version(z::Zypper)
+    uname = readchomp(`uname -m`)
+    found_uname = false
+    found_version = false
+    for l in eachline(`zypper info $(z.package)`)
```
Comment (tkelman) [inline src/dependencies.jl]: ref https://github.com/JuliaLang/BinDeps.jl/issues/188#issuecomment-161597464, I think we want to do something like
```
ENV2 = copy(ENV)
ENV2["LC_ALL"] = "C"
for l in eachline(setenv(`zypper info $(z.package)`, ENV2))
```

And what are `found_uname` and `found_version` being used for here?

**Comment by colbec** [comment]
Oh dear, the variables `found_*` are unintentional residual from testing that should have been removed.
Issue https://github.com/JuliaLang/julia/issues/14249 which has a bearing on the `ENV` variable setting seems to be still ongoing. Proceed anyway?

**Comment by tkelman** [comment]
Whatever the resolution of that issue will be, it will take effect in 0.5 at the earliest. BinDeps.jl still needs to support 0.3 and 0.4 at the moment. I would hope any new features will have deprecation paths, at which point Compat.jl and BinDeps.jl could be updated accordingly, but for now I think you should proceed if you'd like to see this feature in the short term.

**Comment by nalimilan** [comment]
Agreed, that discussion shouldn't block you from adding Zypper support.

**Comment by tkelman** [comment]
I think this is redundant with #193, so I'll close.

**Pull request closed without merging**